src/main/java/com/breskul/bibernate/persistence/Session.java

```diff
@@ -112,6 +112,10 @@ public Transaction getTransaction() {
     return transaction;
   }
 
+  public <T> void delete(T entity) {
+    genericDao.delete(entity);
```

Comment:
are we check if entity is managed?

Reply:
![image](https://github.com/bobocode-breskul/bibernate/assets/60611618/a972930d-6309-49a9-8bb2-a465eedc3728)

We check is the entity in the context. Is it the same?